apps/aecore/lib/aecore/chain/block_validation.ex

```diff
@@ -76,17 +76,26 @@ defmodule Aecore.Chain.BlockValidation do
 
   @spec validate_block_transactions(Block.block()) :: list()
   def validate_block_transactions(block) do
-    block.txs |> Enum.map(fn tx -> SignedTx.is_coinbase(tx) ||  SignedTx.is_valid(tx) end)
+    block.txs
+    |> Enum.map(
+         fn tx -> cond do
+                    SignedTx.is_coinbase(tx) ->
+                      true
+                    true ->
+                      KeyManager.verify(tx.data, tx.signature, tx.data.from_acc)
```

Comment:
why did you revert this to the old validation check?

Reply:
we should use `SignedTx.is_valid(tx)` which should include non negative value and non negative fee checks